test/test_CommandAndStateMessageParser.cpp

```diff
@@ -193,7 +193,7 @@ TEST_F(MessageParserTest, it_parse_tilt_camera_head_command_message)
     command.camera.zoom.speed = 0.75;
     base::samples::Joints tilt;
     JointState joint_state;
-    joint_state.position = M_PI;
+    joint_state.position = 3.0;
```

Comment:
This should have lit a big red light in your head.

"I've just treated a value as degrees but tests set M_PI .... Mmmmmm"